tango/pyutil.py

```diff
@@ -207,6 +207,12 @@ def __Util__init__(self, args):
     Util.__init_orig__(self, args)
 
 
+def __Util__init(args):
+    args = list(args)
+    args[0] = os.path.splitext(args[0])[0]
```

Comment:
@tiagocoutinho Are we dropping the file extension because cppTango 9.3.x doesn't handle it?  I suggest we add a comment here to say why we doing this.

Reply:
Good catch. I have to verify. I will get back to you on this

Reply:
@tiagocoutinho Any feedback on this?